### ISSUE TITLE
[Multi-Tenant]MIResponse.hasError() no longer working in some cases KB 2159861

### ISSUE THREAD

**Issue by PeterVanMunster** [infor-cloud/m3-h5-sdk#75]
**Describe the bug**
In some cases the MIReponse.hasError() method returns false even though an NOK message has been sent.
This is due to a change on the rest api on multi-tenant for list api's see KB 2159861.

**To Reproduce**
1. execute Program: ATS020MI Transaction: SelAttrValue and ATID on a value not in M3.
2. The api response will contain a nok message in content.ErrorMessage instead of on content itself (ParseMessage()).
3. The error fields on MIResponse are now undefined and hasError() will incorrectly return false.

**Solution**
Update the parseMessage method with:
```typescript
private parseMessage(response: IMIResponse, content: any) {
      let code = content['@code'];
      let field = content['@field'];
      let errorType = content['@type'];
      let message = content['Message']; 
      if(content.ErrorMessage) {
          code = content.ErrorMessage["@code"];
          field = content.ErrorMessage["@field"];
          errorType = content.ErrorMessage["@type"];
          message = content.ErrorMessage["Message"];
      }
      if (message) {
         // Clean up the message that might contain the code, field and a lot of whitespace.
         if (code) {
            message = message.replace(code, '');
         }
         if (field) {
            message = message.replace(field, '');
         }
         response.errorMessage = message.trim();
      }
      response.errorCode = code ? code : errorType;
      response.errorField = field;
      response.errorType = errorType;
   }
```

**Comment by anhallbe** [comment]
Hi @PeterVanMunster! Thanks for the report.
A fix has been added to an upcomming patch version (`3.0.2-next.1`). Could you try it out in your project and see if the issue is resolved?

Run this in your existing project:
```
npm i @infor-up/m3-odin@3.0.2-next.1 @infor-up/m3-odin-angular@3.0.2-next.1
```

**Comment by anhallbe** [comment]
Fixed in 3.0.2
